### PR TITLE
{bio} BLAST-2.2.26-Linux_x86_64 update source url

### PR DESCRIPTION
**Pull request by zemu-unile**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — DISMISSED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/e2970026e3de041c20ca91efdaaf82e8 for a full test report.

File: easybuild/easyconfigs/b/BLAST/BLAST-2.2.26-Linux_x86_64.eb [hunk 1]
```diff
@@ -16,7 +16,7 @@ description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
 
 toolchain = SYSTEM
 
-source_urls = ['https://ftp.ncbi.nlm.nih.gov/blast//executables/legacy/%(version)s/']
+source_urls = ['https://ftp.ncbi.nlm.nih.gov/blast/executables/legacy.NOTSUPPORTED/%(version)s/']
 sources = ['%(namelower)s-%(version)s-x64-linux.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BLAST/BLAST-2.2.26-Linux_x86_64.eb:20]: @zemu-unile Please add a SHA256 checksum, `eb --inject-checksums` is very handy for this.

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2640.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1599433828f15bd163a8d8e544bf9359 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Going in, thanks @zemu-unile!

**Pull request merged**